prometheus-to-sd/translator/translator_test.go

```diff
@@ -416,6 +426,136 @@ func TestTranslatePrometheusToStackdriver(t *testing.T) {
 	}
 }
 
+func TestTranslateSummary(t *testing.T) {
+	var intSummaryMetricsResponse = &PrometheusResponse{rawResponse: `
+# TYPE process_start_time_seconds gauge
+process_start_time_seconds 1234567890
+# TYPE int_summary_metric summary
+int_summary_metric{quantile="0.5"} 4
+int_summary_metric{quantile="0.9"} 8
+int_summary_metric{quantile="0.99"} 8
+int_summary_metric_sum 42
+int_summary_metric_count 101010
+`}
+	var floatSummaryMetricsResponse = &PrometheusResponse{rawResponse: `
+# TYPE process_start_time_seconds gauge
+process_start_time_seconds 1234567890
+# TYPE float_summary_metric summary
```

Comment:
Please add one test case with multiple labels, to ensure that no one removes code once you add it.

Reply:
will do.